packages/twenty-ui/src/theme/constants/SecondaryColors.ts

```diff
@@ -10,6 +10,7 @@ export const SECONDARY_COLORS = {
   yellow30: '#ffedaf',
   yellow20: '#fff6d7',
   yellow10: '#fffbeb',
+  yellowIcon: '#ffd338',
```

Comment:
Unsure about naming

Reply:
@Bonapara

Reply:
We should use the colors from: 

https://github.com/twentyhq/twenty/blob/1490c5bd4e5737f7568ae5aa5001169abbccc15c/packages/twenty-ui/src/theme/constants/MainColors.ts